File: app/src/main/java/com/shahu/weathery/model/forecast/ForecastListItem.java
package com.shahu.weathery.model.forecast;

import java.util.List;
import javax.annotation.Generated;
import com.google.gson.annotations.SerializedName;
import com.shahu.weathery.model.common.WeatherItem;

@Generated("com.robohorse.robopojogenerator")
public class ForecastListItem {

	@SerializedName("dt")
	private long dt;

	@SerializedName("dt_txt")
	private String dtTxt;

	@SerializedName("weather")
	private List<WeatherItem> weather;

	@SerializedName("main")
	private Main main;

	@SerializedName("clouds")
	private Clouds clouds;

	@SerializedName("sys")
	private Sys sys;

	@SerializedName("wind")
	private Wind wind;

	@SerializedName("rain")
	private Rain rain;

	public void setDt(long dt){
		this.dt = dt;
	}

	public long getDt(){
		return dt;
	}

	public void setDtTxt(String dtTxt){
		this.dtTxt = dtTxt;
	}

	public String getDtTxt(){
		return dtTxt;
	}

	public void setWeather(List<WeatherItem> weather){
		this.weather = weather;
	}

	public List<WeatherItem> getWeather(){
		return weather;
	}

	public void setMain(Main main){
		this.main = main;
	}

	public Main getMain(){
		return main;
	}

	public void setClouds(Clouds clouds){
		this.clouds = clouds;
	}

	public Clouds getClouds(){
		return clouds;
	}

	public void setSys(Sys sys){
		this.sys = sys;
	}

	public Sys getSys(){
		return sys;
	}

	public void setWind(Wind wind){
		this.wind = wind;
	}

	public Wind getWind(){
		return wind;
	}

	public void setRain(Rain rain){
		this.rain = rain;
	}

	public Rain getRain(){
		return rain;
	}

	@Override
 	public String toString(){
		return 
			"ForecastListItem{" +
			"dt = '" + dt + '\'' + 
			",dt_txt = '" + dtTxt + '\'' + 
			",weather = '" + weather + '\'' + 
			",main = '" + main + '\'' + 
			",clouds = '" + clouds + '\'' + 
			",sys = '" + sys + '\'' + 
			",wind = '" + wind + '\'' + 
			",rain = '" + rain + '\'' + 
			"}";
		}
}

File: app/src/main/java/com/shahu/weathery/model/forecast/ForecastListItem.kt
package com.shahu.weathery.model.forecast

import com.google.gson.annotations.SerializedName
import com.shahu.weathery.model.common.WeatherItem
import javax.annotation.Generated

@Generated("com.robohorse.robopojogenerator")
class ForecastListItem {
    @SerializedName("dt")
    var dt: Long = 0

    @SerializedName("dt_txt")
    var dtTxt: String? = null

    @SerializedName("weather")
    var weather: List<WeatherItem>? = null

    @SerializedName("main")
    var main: Main? = null

    @SerializedName("clouds")
    var clouds: Clouds? = null

    @SerializedName("sys")
    var sys: Sys? = null

    @SerializedName("wind")
    var wind: Wind? = null

    @SerializedName("rain")
    var rain: Rain? = null

    override fun toString(): String {
        return "ForecastListItem{" +
                "dt = '" + dt + '\'' +
                ",dt_txt = '" + dtTxt + '\'' +
                ",weather = '" + weather + '\'' +
                ",main = '" + main + '\'' +
                ",clouds = '" + clouds + '\'' +
                ",sys = '" + sys + '\'' +
                ",wind = '" + wind + '\'' +
                ",rain = '" + rain + '\'' +
                "}"
    }
}